tests/depth_average_06.prm

```diff
@@ -0,0 +1,90 @@
+# Test the 'depth average' postprocessor on a coarse mesh version of
+# convection_box.prm. This test makes no sense since we average over 10 depth
+# intervals but we only compute on a 4x4 mesh -- This is sufficient to produce
+# reasonable data, but not enough for actual resolution.
```

Comment:
Can you update the documentation?
This is a copy of test..., with the only difference that...
(or even better, just include the other input file and then only list your changes, if there aren't that many)